Community Rules/nitro_winner_template.md

```
@@ -0,0 +1,45 @@
+**NITRO BOOST AGREEMENT**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)